toolkit/tools/imagecustomizerapi/module.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+)
+
+type Modules struct {
+	Load   []Module `yaml:"Load"`
```

Comment:
Should we explicitly call it "Blacklist", so it sets the right expectations?

Reply:
The word "blacklist" should generally be avoided. Substitutions include "block list", "deny list", and "disallow list".

(Though there isn't much that can be done when using an existing API, such as `/etc/modprobe.d/`.)

Reply:
I hear you.. but then I thought that will be the best representation of what we do under the hoods and all related implications that comes with modprobe implementation. 
Maybe that's something we can clarify instead in our documentation